chromium_src/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc

```diff
@@ -285,6 +285,10 @@ void BraveAddCommonStrings(content::WebUIDataSource* html_source,
       IDS_SETTINGS_HANGOUTS_ENABLED_DESC},
     {"resolveIPFSURLDesc",
       IDS_SETTINGS_RESOLVE_IPFS_URLS_DESC},
+    {"ipfsAutoFallbackToGatewayLabel",
+      IDS_SETTINGS_IPFS_AUTO_FALLBACK_TO_GATEWAY_LABEL},
+    {"ipfsAutoFallbackToGatewayDesc",
+      IDS_SETTINGS_IPFS_AUTO_FALLBACK_TO_GATEWAY_DESC},
```

Comment:
cc @bridiver here for chromium_Src owner review.

Reply:
No need for @bridiver to review this particular chromium_src change. The codeowner file has some false positives in some cases like this which doesn't need his review.